ui/app/models/structured-attributes.js

```diff
@@ -8,6 +8,11 @@ const { unflatten } = flat;
 export default class StructuredAttributes extends Fragment {
   @attr() raw;
 
+  recomputeRawProperties(incoming) {
+    this.set('raw', incoming);
```

Comment:
Calling `set` means we'll have to use the `classic` decorator here which means we're not Octane compatible.

The [docs](https://guides.emberjs.com/release/models/creating-updating-and-deleting-records/) are saying we might be able to get away with dot notation.

Reply:
As in `this.raw = {...incoming}`

Reply:
`this.raw = incoming` seems to work so going with that, without object deconstruction cloning